charts/memgraph-high-availability/templates/data.yaml

```diff
@@ -151,6 +151,9 @@ spec:
             value: {{ $.Values.env.MEMGRAPH_ENTERPRISE_LICENSE }}
           - name: MEMGRAPH_ORGANIZATION_NAME
             value: {{ $.Values.env.MEMGRAPH_ORGANIZATION_NAME }}
+          {{- with $.Values.extraEnv.data }}
+          {{- toYaml . | nindent 10 }}
```

Comment:
the installation worked for nindent 12, not 10

Reply:
set to 12